npm/vite-plugin-cypress-esm/src/index.ts

```diff
@@ -71,24 +72,21 @@ export const CypressEsm = (options?: CypressEsmOptions): Plugin => {
   /**
    * If a given import target (path) is explicitly ignored then bypass our custom mapping on it
    *
-   * @param importTarget
+   * @param sanitizedImportTarget
    * @returns
    */
-  const isImportOnIgnoreList = (importTarget: string) => {
-    return ignoreImportMatcher(importTarget)
+  const isImportOnIgnoreList = (sanitizedImportTarget: string) => {
+    return ignoreImportMatcher(sanitizedImportTarget)
```

Comment:
The import ignore matcher didn't work correctly. Now it does, after removing the quotes and semicolon from the import target.

Reply:
I further improved the regex to only capture what's between the quotes.